src/lib/get-input-value.ts

```diff
@@ -0,0 +1,7 @@
+import { ChangeEvent } from 'react';
```

Comment:
Please, name each library as entity or process. "get" or "set" or "add" is a verb prefix, used to name functions. Lib is not a function, it is a collection of functions, even if it is only one function here. How can you name collection of functions?

Reply:
Renamed to `input.ts`.